daemon/delete.go

```diff
@@ -165,3 +161,11 @@ func (daemon *Daemon) commonRm(container *Container, forceRemove bool) (err erro
 
 	return nil
 }
+
+func toBool(str string) bool {
```

Comment:
@LK4D4 this basically do the same as https://github.com/docker/docker/blob/master/engine/env.go#L57
but that one use `env` to get the key etc etc...I don't like this toBool here... should I move it elsewhere? (where)